### PR TITLE
[terra-action-footer] - Restructure theme tests

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary
Moved theme tests into a separate file and theme directory.

### Additional Details
Part of #2006 

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  withoutActionsThemeables,
+} = require('../common');
+
+describe('Action footer theme', () => {
+  describe('Empty', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-action-footer/action-footer/empty-action-footer'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty(withoutActionsThemeables);
```
Comment (emilyrohrbough) [inline packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js]: We should be able to drop 
```
Terra.should.beAccessible({ viewports });
Terra.should.matchScreenshot({ viewports });
```
right?

File: packages/terra-action-footer/tests/wdio/action-footer-spec.js [hunk 2]
```diff
@@ -1,16 +1,11 @@
-const {
-  viewports,
-  withActionsThemeables,
-  withoutActionsThemeables,
-} = require('./common');
+const { viewports } = require('./common');
 
 describe('ActionFooter', () => {
   describe('Empty', () => {
```
Comment (emilyrohrbough) [inline packages/terra-action-footer/tests/wdio/action-footer-spec.js]: We are in here restructuring tests, does capturing a screenshot and testing accessibility of an empty action footer add value or can we drop this?

File: packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js (hunk 1 above)
Comment (StephenEsser) [reply packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js]: We could, I left the tests exactly as they were just to ensure these test changes were passive in what they were targeting

File: packages/terra-action-footer/tests/wdio/action-footer-spec.js (hunk 2 above)
Comment (StephenEsser) [reply packages/terra-action-footer/tests/wdio/action-footer-spec.js]: Are there any strong opinions for keeping or removing it?

File: packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js (hunk 1 above)
Comment (StephenEsser) [reply packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js]: I went ahead and removed these from the theme tests in previous commits,  I think Github is previewing an old commit this thread is on.

https://github.com/cerner/terra-core/pull/2021/commits/ce4c52b6f3bff6eb75c0d111dd4495c6c89ba596

https://github.com/cerner/terra-core/pull/2021/commits/49a2f11b89dba323f140a8728c14bc0d868ddc96

Comment (emilyrohrbough) [reply packages/terra-action-footer/tests/wdio/theme/action-footer-theme-spec.js]: whoops 😄

File: packages/terra-action-footer/tests/wdio/action-footer-spec.js (hunk 2 above)
Comment (bjankord) [reply packages/terra-action-footer/tests/wdio/action-footer-spec.js]: Let's remove it.

Comment (StephenEsser) [reply packages/terra-action-footer/tests/wdio/action-footer-spec.js]: Removed: https://github.com/cerner/terra-core/pull/2021/commits/7300cbb728738327a20cb1423ad56c86284efdc6

**Pull request merged**